applications/kodadot_assethub_nft_indexer_statemine_statemint.md

```diff
@@ -0,0 +1,210 @@
+| 7d.   | On-chain Attributes CLEAR Handler    | Developing a handler to index clearance of the attribute for the collection from the chain.                   |
+| 7e.   | On-chain Attributes CLEAR for NFT    | Implementation of the indexing handler to clear an attribute for NFT from the chain.                          |
+| 7f.   | Metadata CREATE Handler                    | Deployment of an indexing handler to set metadata for the collection from the chain.                          |
+| 7g.   | Metadata CREATE for NFT                     | Creating an indexing handler to set metadata for NFT from the chain.                                          |
+| 7h.   | Metadata CLEAR Handler                      | Developing a handler to index clearance of metadata for the collection from the chain.                        |
+| 7i.   | Metadata CLEAR for NFT                       | Introducing a handler for indexing clearance of metadata for NFT from the chain.                              |
+| 7j.   | Metadata IPFS Integration Handler     | Design a handler to retrieve IPFS Metadata from the IPFS network.                                                 |
+| 7l.   | Metadata Schema Development                | Creating and designing a GraphQL schema entity representing metadata.                                     |
+| 8a.   | NFT Royalties Schema Integration      |Design and include royalty support within the GraphQL schema.                                                  |
+| 8b.   | NFT Royalties Addition Handler        | Implement a handler to add royalty into NFT.                                                 |
+| 8c.   | NFT Royalties PAYOUT Handler           |Creation of handler to index royalty payout events from the chain.                                             |
+| 9.      | Fungible Assets Schema Creation      |Design and formulation of a GraphQL schema entity representing fungible assets.                                |
+| 9.      | Fungible Assets Force Creation        |Handlers will be developed to add system tokens like KSM/DOT into fungible assets.                             |
+| 9.      | Fungible Assets CREATE Event          |An event handler for indexing the creation of a fungible event from a chain, such as (RMRK/USDT) will be developed.|
+| 9.      | Metadata Support for Fungible Assets | Implement a handler to add metadata to a fungible asset event from the chain.                          |
+| 9c.   | Fungible Asset Allowlist Setup        |Setting up allows list-based indexing of fungible assets.                                                       |
+| 10a.  | Data Views Development                |Construction of data views for efficient access to indexed data.                                               |
+| 10b.  | Implementing Metadata Caching Layer |Develop and retry IPFS metadata if un-indexed by 7j.                                                               |
```

Comment:
What does 'unindexed' mean here? Unpinned?

Reply:
I think this means that we will store those data if particular IPFS hashes are not pinned by original author for better UX.